lib/client.js

```diff
@@ -902,9 +922,17 @@ Request.prototype.end = function(fn){
   // set header fields
   for (var field in this.header) {
     if (null == this.header[field]) continue;
+    // blob and arraybuffer requests use responseType, not header Content-Type
+    // https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Sending_and_Receiving_Binary_Data    
+    if (field === 'Content-Type' && this.responseType) continue;
```

Comment:
Thanks for pointing this out. This condition exists somewhat inadvertently as it prevents cycle-http-driver (which I am using) from using a default "json" `Content-Type` when responseType is defined. I'll will remove the condition before committing again.

https://github.com/cyclejs/cycle-http-driver/blob/master/src/http-driver.js#L16